shell/platform/tizen/tizen_renderer.h

```diff
@@ -36,6 +37,7 @@ class TizenRenderer {
   virtual void SetRotate(int angle) = 0;
   virtual void ResizeWithRotation(int32_t x, int32_t y, int32_t width,
                                   int32_t height, int32_t degree) = 0;
+  virtual void SetPreferredOrientations(std::vector<int> rotations) = 0;
```

Comment:
```suggestion
  virtual void SetPreferredOrientations(const std::vector<int>& rotations) = 0;
```

If you agree to change this, please remember to change signatures of definitions of this method.